app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -3,8 +3,9 @@
     <div :class="wrapClass">
       <div v-tooltip.top-start="sentByMessage" :class="bubbleClass">
         <bubble-mail-head
-          v-if="isEmailContentType"
           :email-attributes="contentAttributes.email"
+          :cc="contentAttributes.cc_emails"
```

Comment:
```suggestion
          :cc="emailHeadAttributes.cc"
```

Reply:
@nithindavid Actually, this is how we are getting and storing the cc_emails and bcc_emails value that we are getting in the content_attributes json https://github.com/chatwoot/chatwoot/pull/3098/files#diff-475917f5423fc3a4e4089f7f8e7a12c1b5136f6c030485f7f602cc44f8b32995R44